tests/test_app/templates/base.html

```diff
@@ -1,23 +1,22 @@
 {% load static %} {% load idom %}
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="shortcut icon"
-      type="image/png"
-      href="{% static 'favicon.ico' %}"
-    />
-    <title>IDOM</title>
-  </head>
 
-  <body>
-    <h1>IDOM Test Page</h1>
-    <div>{% idom_component "test_app.components.HelloWorld" class="hello-world" %}</div>
-    <div>{% idom_component "test_app.components.Button" class="button" %}</div>
-    <div>{% idom_component "test_app.components.ParametrizedComponent" class="parametarized-component" x=123 y=456 %}</div>
-    <div>{% idom_component "test_app.components.SimpleBarChart" class="simple-bar-chart" %}</div>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="shortcut icon" type="image/png" href="{% static 'favicon.ico' %}" />
+  <title>IDOM</title>
+</head>
+
+<body>
+  <h1>IDOM Test Page</h1>
+  <div>{% idom_component "test_app.components.HelloWorld" class="hello-world" %}</div>
+  <div>{% idom_component "test_app.components.Button" class="button" %}</div>
+  <div>{% idom_component "test_app.components.ParametrizedComponent" class="parametarized-component" x=123 y=456 %}
+  </div>
+  <div>{% idom_component "test_app.components.SimpleBarChart" class="simple-bar-chart" %}</div>
+</body>
+
+</html>
```

Comment:
missing end of file new line

Reply:
On the note of EOF new lines, I'm going to make a PR to both DJI and Core to add a vscode configuration to automate things like this for new devs passing by.